tt_eager/tt_dnn/op_library/sharded/kernels/dataflow/reader_unary_stick_layout_sharded_blocks_interleaved_start_id.cpp

```diff
@@ -7,57 +7,83 @@
 
 void kernel_main() {
 
-    const uint32_t src_addr                 = get_arg_val<uint32_t>(0);
```

Comment:
What kind of code formatter did you use on these kernel files? We normally use clang-format if we format code and this doesn't seem to match.